config/kibana.yml

```diff
@@ -180,4 +180,4 @@
 
 # Maximum number of documents loaded by each shard to generate autocomplete suggestions.
 # This value must be a whole number greater than zero. Defaults to 100_000
-#unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
+#unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
```

Comment:
I think if you back out this new line, the kibana core CODEOWNERS requirement will go away?